Makefile

```diff
@@ -30,6 +30,21 @@ test:
 .PHONY: build
 build: format proto tools
 
+.PHONY: docker_build
+docker_build: build
+	docker build -t gcr.io/config-validator/config-validator:latest .
+
+.PHONY: docker_run
+docker_run: guard-POLICY_LIBRARY_DIR docker_build
```

Comment:
Add a comment regarding usage for this make rule

Reply:
Added some comments to this makefile.